tests/core/test_channels.py

```diff
@@ -369,6 +370,75 @@ def test_socketio_channel():
     assert routes_list["handle_request"].startswith("/socket.io")
 
 
+async def test_socketio_channel_jwt_authentication():
+    import jwt
+    from rasa.core.channels.channel import _decode_bearer_token
+    from rasa.core.channels.socketio import SocketIOInput
```

Comment:
why do we need local imports? We should always use global imports except to avoid cyclic imports or performance reasons